packages/airnode-utilities/src/gas-prices/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './gas-prices';
```

Comment:
I wonder if this should be in an "evm" folder? These functions/types are EVM specific